### PR TITLE
#164798019 User can delete article

### PR DESCRIPTION
**Pull request by sojida**
#### Description
This PR is a route for the user to be able to delete an article
 
#### Type of change
- [ ] New feature(Delete article route)

#### How Has This Been Tested?
- [ ] Unit Testing

#### Checklist:
- [ ] write test cases
- [ ] create delete article route

#### Screenshots
Screenshot 2019-04-05 at 7 40 54 PM

### CODE REVIEW

File: server/controllers/article.controllers.js [hunk 1]
```diff
@@ -55,6 +55,36 @@ const controller = {
...
+        });
+      }
+
+      return res.status(200).json({
+        message: 'Article Deleted Successfully',
+      });
+    } catch (error) {
+      return res.status(500).json({
+        error,
+        message: 'Oops! There seem to be a database error',
```
Comment (rotimi-babalola) [inline server/controllers/article.controllers.js]: Nitpick: `... seems to be ...`

File: server/routes/article.routes.js [hunk 2]
```diff
@@ -6,5 +6,6 @@ const { articleController } = controllers;
 const router = express.Router();
 
 router.get('/article/:id', articleController.getOneArticle);
+router.delete('/article/:id', articleController.deleteArticle);
```
Comment (rotimi-babalola) [inline server/routes/article.routes.js]: Will there be a future PR to secure these routes?

File: server/controllers/article.controllers.js (hunk 1 above)
Comment (sojida) [reply server/controllers/article.controllers.js]: 👍🏾

File: server/routes/article.routes.js (hunk 2 above)
Comment (sojida) [reply server/routes/article.routes.js]: Yes. I would update it while I try to fix this merge conflict.

File: server/routes/article.routes.js [hunk 3]
```diff
@@ -12,4 +12,10 @@ router.get(
   articleController.getOneArticle
 );
 
+router.delete(
+  '/article/:id',
+  validations.verifyToken,
+  articleController.deleteArticle
+);
+
```
Comment (rotimi-babalola) [inline server/routes/article.routes.js:30]: Check that only owners of an article can delete it.

**Review by rotimi-babalola — CHANGES_REQUESTED** [review]

**Pull request merged**